### PR TITLE
2.1.1 release notes

### PR DESCRIPTION
**Pull request by danielmewes**
@mglukhovsky could you review please?

The fix for #4664 probably won't make it in. I'll remove it from the release notes before merging in that case.

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+# Release 2.1.1 (Forbidden Planet)
+
+Released on 2015-08-12
+
+Bug fix release
+
+* Fixed a problem that made replicas get stuck when reconfiguring a table after migration
```
Comment (mglukhovsky) [inline NOTES.md]: Fixed a bug where replicas don't become available when reconfiguring a table after migration.

Comment (mglukhovsky) [inline NOTES.md]: Correction:

> Fixed a problem: after migration, some replicas were unavailable when reconfiguring a table

**Comment by mglukhovsky** [comment]
@dalanmiller one comment, but looks good!

**Comment by danielmewes** [comment]
@mglukhovsky Thanks, updated the draft.

**Comment by mglukhovsky** [comment]
@dalanmiller :white_check_mark:  approved

**Pull request merged**